client/state/purchases/index.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * External Dependencies
+ */
+import assign from 'lodash/assign';
+import find from 'lodash/find';
+import matches from 'lodash/matches';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	PURCHASES_REMOVE,
```

Comment:
Actually, why did you change this from just `
import { action as ActionTypes } from 'lib/upgrades/constants';`?